internal/status/http-statuses.csv

```diff
@@ -40,6 +40,9 @@
 416,4,Requested Range Not Satisfiable,RFC7231
 417,4,Expectation Failed,RFC7231
 418,4,I'm a teapot,RFC2324
+419,4,Page Expired,Unofficial
```

Comment:
Instead of `Unofficial`, could you give the technology using it please?

Reply:
I've amended the PR. There is however one issue. For codes 598 and 599 I could not find a specific technology that is using it. Wikipedia refers to this one [wayback machine page for ascii-code.com](https://web.archive.org/web/20170107115008/http://www.ascii-code.com/http-status-codes.php) where it says those codes are `Informal convention`.

Should I:

- Leave as `Unofficial`
- Change it to `Informal Convention`
- Remove the codes altogether

?

Reply:
> I've amended the PR
Thx.

> For codes 598 and 599
I will change to `Unknown` as we don't have any sources...